### ISSUE TITLE
Featured Image Not Cropped on Home Page after Cropping

### ISSUE THREAD

**Issue by markolbert** [vollyimnetz/crop-thumbnails#43]
I can't seem to get the plugin to work. Or, rather, the cropping process works fine on featured images...but the results don't display on my home page, where the posts are listed.

I have done ctrl-F5 to refresh the cache. I'm using Avatar as a theme with Elementor for layout control.

Thoughts?

**Comment by vollyimnetz** [comment]
If the images are cropped in backend, make sure they are really used in frontend. Depending on your theme there may be using uncropped image sizes.

**Comment by eursula** [comment]
Hi, I am getting the same issue... the images don't seem to update, yet they're cropped in the admin section. I have to do a backend server cache flush to push them through. I'm sure this has only started happening recently though. I've been looking to see if there's any caching info, can you confirm anything on this? Thanks

**Comment by vollyimnetz** [comment]
How to check?
1. Are the cropped image sizes used in the frontend -> check by right-click on the image -> load in new tab -> have a look on the image name if its ending with 1234x456 (your image-size) then the theme uses your croppend image and then got to second point. If the image do not use the croppend image size, then your theme do not use the image size there. Blame your theme.
2. If the cropped image is loaded in the frontend but crop changes are not visible then you may hard reload your browser. See faq of the plugin -> https://wordpress.org/plugins/crop-thumbnails/#i%E2%80%99ve%20cropped%20the%20image%2C%20but%20the%20new%20version%20do%20not%20appear%20in%20the%20frontend.

**Comment by eursula** [comment]
Thanks for your reply. Yup, the image is def cropping the correct dimensions, but it's the viewbox section that isnt being captured. Tried the caching thing, but still not working.. I've looked at if it's conflicting with any other plugins, deactivated some others but it's not making any difference. Here's the site if you want to take a look ? This page in particular: https://www.evolutionequestrian.co.nz/news-blogs/. Thanks
